web/assets/js/Utils.js

```diff
@@ -0,0 +1,6 @@
+class Utils {
```

Comment:
This class also looks like something not needed. Can be done with an object.

Reply:
Also defining helper like this in global scope might be risky. Easy for name collision.